components/developer/meld/meld.p5m

```diff
@@ -37,80 +36,197 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file path=usr/lib/meld/meld/__init__.py
-file path=usr/lib/meld/meld/dbus_service.py
-file path=usr/lib/meld/meld/diffmap.py
-file path=usr/lib/meld/meld/diffutil.py
-file path=usr/lib/meld/meld/dirdiff.py
-file path=usr/lib/meld/meld/filediff.py
-file path=usr/lib/meld/meld/filemerge.py
-file path=usr/lib/meld/meld/filters.py
-file path=usr/lib/meld/meld/linkmap.py
-file path=usr/lib/meld/meld/matchers.py
-file path=usr/lib/meld/meld/meldapp.py
-file path=usr/lib/meld/meld/meldbuffer.py
-file path=usr/lib/meld/meld/melddoc.py
-file path=usr/lib/meld/meld/meldwindow.py
-file path=usr/lib/meld/meld/merge.py
-file path=usr/lib/meld/meld/misc.py
-file path=usr/lib/meld/meld/newdifftab.py
-file path=usr/lib/meld/meld/patchdialog.py
-file path=usr/lib/meld/meld/paths.py
-file path=usr/lib/meld/meld/preferences.py
-file path=usr/lib/meld/meld/recent.py
-file path=usr/lib/meld/meld/task.py
-file path=usr/lib/meld/meld/test_matchers.py
-file path=usr/lib/meld/meld/tree.py
-file path=usr/lib/meld/meld/ui/__init__.py
-file path=usr/lib/meld/meld/ui/emblemcellrenderer.py
-file path=usr/lib/meld/meld/ui/findbar.py
-file path=usr/lib/meld/meld/ui/gladesupport.py
-file path=usr/lib/meld/meld/ui/gnomeglade.py
-file path=usr/lib/meld/meld/ui/historyentry.py
-file path=usr/lib/meld/meld/ui/listwidget.py
-file path=usr/lib/meld/meld/ui/msgarea.py
-file path=usr/lib/meld/meld/ui/notebooklabel.py
-file path=usr/lib/meld/meld/ui/statusbar.py
-file path=usr/lib/meld/meld/ui/vcdialogs.py
-file path=usr/lib/meld/meld/ui/wraplabel.py
-file path=usr/lib/meld/meld/undo.py
-file path=usr/lib/meld/meld/util/__init__.py
-file path=usr/lib/meld/meld/util/compat.py
-file path=usr/lib/meld/meld/util/prefs.py
-file path=usr/lib/meld/meld/util/sourceviewer.py
-file path=usr/lib/meld/meld/vc/__init__.py
-file path=usr/lib/meld/meld/vc/_null.py
-file path=usr/lib/meld/meld/vc/_vc.py
-file path=usr/lib/meld/meld/vc/bzr.py
-file path=usr/lib/meld/meld/vc/cvs.py
-file path=usr/lib/meld/meld/vc/darcs.py
-file path=usr/lib/meld/meld/vc/fossil.py
-file path=usr/lib/meld/meld/vc/git.py
-file path=usr/lib/meld/meld/vc/mercurial.py
-file path=usr/lib/meld/meld/vc/monotone.py
-file path=usr/lib/meld/meld/vc/svk.py
-file path=usr/lib/meld/meld/vc/svn.py
-file path=usr/lib/meld/meld/vc/svn_17.py
-file path=usr/lib/meld/meld/vcview.py
-file path=usr/share/appdata/meld.appdata.xml
-file path=usr/share/applications/meld.desktop
-file path=usr/share/gnome/help/meld/C/meld.xml
-file path=usr/share/gnome/help/meld/es/meld.xml
-file path=usr/share/gnome/help/meld/fr/meld.xml
-file path=usr/share/icons/HighContrast/scalable/apps/meld.svg
-file path=usr/share/icons/hicolor/16x16/apps/meld.png
-file path=usr/share/icons/hicolor/22x22/apps/meld.png
-file path=usr/share/icons/hicolor/32x32/apps/meld.png
-file path=usr/share/icons/hicolor/48x48/apps/meld.png
-file path=usr/share/icons/hicolor/scalable/apps/meld.svg
+# Bypass Tk dependencies, since they're only used to display an error dialog
```

Comment:
I think you should add the runtime dependencies here like I did for ansible. Ansible's runtime dependencies (it uses python-3.7 atm) look like this:
depend type=require fmri=library/python/cryptography-37
depend type=require fmri=library/python/jinja2-37
depend type=require fmri=library/python/paramiko-37
depend type=require fmri=library/python/pyasn1-37
depend type=require fmri=library/python/pyyaml-37
depend type=require fmri=library/python/six-37

Note: Depending on a python meta-package doesn't necessarily bring the package version that is needed. It just installs all available versions but the version for the actual python version used doesn't need to be amoung them.
This was the case for some packages ansible depend on and can only be found by testing or using the application.

Reply:
Did it, thanks. I skipped `library/python/pygobject-3-39` because it's automatically detected as dependency and pkglint gave me an error of duplicate dependency.